mexico-city/src/managers/buffer.rs

```diff
@@ -0,0 +1,196 @@
+use err_derive::Error;
```

Comment:
Is there a header in this file?  I can't tell from the Github user-interface.

Reply:
I just added one.